tests/appsec/test_automated_user_and_session_tracking.py

```diff
@@ -244,6 +249,7 @@ def setup_session_blocking(self):
             cookies=self.r_create_session.cookies,
         )
 
+    @missing_feature(context.library == "dotnet", reason="Session ids can't be set.")
```

Comment:
maybe this should be in the manifest directly ? also wdym by sessons cant be set ? that shouldn't prevent the test from passing ?

Reply:
I wanted to add a comment to explain why this can't be activated. Also, I'm not sure this test is relevant and maybe it should be deleted as session blocking is not going to be a feature. 
Also, we can't set the session id in dotnet, it's automatically generated by the framework, so we can't predict it